dexbot/strategies/config_parts/relative_config.py

```diff
@@ -32,38 +32,38 @@ def configure(cls, return_base_config=True):
                           'Use external reference price instead of center price acquired from the market', None),
             ConfigElement('external_price_source', 'choice', EXCHANGES[0][0], 'External price source',
                           'The bot will try to get price information from this source', EXCHANGES),
-            ConfigElement('amount', 'float', 1, 'Amount',
+            ConfigElement('amount', 'float', 4.5, 'Amount',
                           'Fixed order size, expressed in quote asset, unless "relative order size" selected',
                           (0, None, 8, '')),
-            ConfigElement('relative_order_size', 'bool', False, 'Relative order size',
+            ConfigElement('relative_order_size', 'bool', True, 'Relative order size',
                           'Amount is expressed as a percentage of the account balance of quote/base asset', None),
             ConfigElement('spread', 'float', 5, 'Spread',
                           'The percentage difference between buy and sell', (0, 100, 2, '%')),
             ConfigElement('dynamic_spread', 'bool', False, 'Dynamic spread',
                           'Enable dynamic spread which overrides the spread field', None),
-            ConfigElement('market_depth_amount', 'float', 0, 'Market depth',
+            ConfigElement('market_depth_amount', 'float', 99, 'Market depth',
                           'From which depth will market spread be measured? (QUOTE amount)',
                           (0.00000001, 1000000000, 8, '')),
-            ConfigElement('dynamic_spread_factor', 'float', 1, 'Dynamic spread factor',
+            ConfigElement('dynamic_spread_factor', 'float', 80, 'Dynamic spread factor',
                           'How many percent will own spread be compared to market spread?',
                           (0.01, 1000, 2, '%')),
             ConfigElement('center_price', 'float', 0, 'Center price',
                           'Fixed center price expressed in base asset: base/quote', (0, None, 8, '')),
             ConfigElement('center_price_dynamic', 'bool', True, 'Measure center price from market orders',
                           'Estimate the center from closest opposite orders or from a depth', None),
-            ConfigElement('center_price_depth', 'float', 0, 'Measurement depth',
+            ConfigElement('center_price_depth', 'float', 999, 'Measurement depth',
                           'Cumulative quote amount from which depth center price will be measured',
                           (0.00000001, 1000000000, 8, '')),
             ConfigElement('center_price_from_last_trade', 'bool', False, 'Last trade price as new center price',
                           'This will make orders move by half the spread at every fill', None),
-            ConfigElement('center_price_offset', 'bool', False, 'Center price offset based on asset balances',
+            ConfigElement('center_price_offset', 'bool', True, 'Center price offset based on asset balances',
```

Comment:
I think it's better to leave this to the user.

Reply:
I think it is better to leave this to the users with 'true', because false makes no sense for RO strategy.

Reply:
It's really depends on how it's being used.